from data.data import Datos
import custom_exceptions

class DatosEstimacion(Datos):
    pass